Multi-Step Form + Bugfix Style Select JavaScript

wisam-alhennawi opened the pull request:
- Multi-Step form: a new feature has been added to build a multi step form.
- Bugfix: styled select component (Javascript error if it used more than one times in the same form)

jpbu reviewed (commented):
Thanks, I reviewed the code changes and tested both a normal tab and multi-step locally with a test app, worked well.

In addition to the inline comments, the following points:
* A description in the README ideally with an example would be very helpful
* It looks to me like the `MultiStepForm` trait only works if the `HasTabs` trait is used, so I would rather add the `use MultiStepForm` to the `HasTabs` than the general `LaraFormsBuilder`
* Styling of tabs in the multi-step case should be changed to not imply that the tabs are clickable/navigateable
* With `confirmBeforeSubmit` set to true, "Previous step" navigation on the confirmation view should reset the confirm mode back to an editable view (in my test, it was just showing read only and there was no way to edit it)

wisam-alhennawi commented on src/LaraFormsBuilder.php:
```diff
@@ -349,17 +360,21 @@ public function switchToEditMode()
+        if ($this->customSuccessMessage != '') {
+            $message = $this->customSuccessMessage;
+        } else {
+            // if you want to customize the success message, you should add the custom message key entry to the lang file (Example: "A new forestryPoolMember has benn created successfully.") or override the method
```
You're right, I'll add it.

wisam-alhennawi commented:
> Thanks, I reviewed the code changes and tested both a normal tab and multi-step locally with a test app, worked well.
> 
> In addition to the inline comments, the following points:
> 
> * A description in the README ideally with an example would be very helpful
> * It looks to me like the `MultiStepForm` trait only works if the `HasTabs` trait is used, so I would rather add the `use MultiStepForm` to the `HasTabs` than the general `LaraFormsBuilder`
> * Styling of tabs in the multi-step case should be changed to not imply that the tabs are clickable/navigateable
> * With `confirmBeforeSubmit` set to true, "Previous step" navigation on the confirmation view should reset the confirm mode back to an editable view (in my test, it was just showing read only and there was no way to edit it)

Thanks for your review and comments. 
Regarding a description in the README, you are right and it is already in my ToDos but I didn't have time to do it yesterday :).
I will add it and implement the other points.

wisam-alhennawi commented:
@jpbu Thank you again for your comments, I implemented them. 
> Styling of tabs in the multi-step case should be changed to not imply that the tabs are clickable/navigateable

In this context I only distinguished the navigation part. We can also split` tabs.blade.php` into two files (tabs and steps) if needed in the future

jpbu reviewed (approved):
Thanks, I double checked the code changes and tested locally with a test app, looks good. Ready to merge from my perspective.
